luatest/assertions.lua

```diff
@@ -118,7 +118,7 @@ end
 --
 -- @string message
 function M.skip(message)
-    utils.luatest_error('skip', message, 2)
+    utils.luatest_error('skip', message or 'No reason', 2)
```

Comment:
```suggestion
    utils.luatest_error('skip', message or '(no reason specified)', 2)
```

There's a slight difference:

* This test was skipped for no reason
* This test was skipped, but no reason was specified

Reply:
Fixed.